compiler/rustc_error_codes/src/lib.rs

```diff
@@ -515,7 +515,6 @@ E0769: 0769,
 E0770: 0770,
 E0771: 0771,
 E0772: 0772,
-E0773: 0773,
```

Comment:
below there's a list of retired error codes (often with a reason). Same for the fact that I've seen examples of error code descriptions not being removed but simply deprecated. This last thing seems unnecessary as this was unstable anyway, but could be nice to list the error code at the bottom as removed.